tests/runtest.sh

```diff
@@ -490,7 +495,11 @@ function run_test {
 }
 
 # Variables for running tests in the background
-((maxProcesses = $(getconf _NPROCESSORS_ONLN) * 3 / 2)) # long tests delay process creation, use a few more processors
+if [ `uname` = "NetBSD" ]; then
+    ((maxProcesses = $(getconf NPROCESSORS_ONLN) * 3 / 2)) # long tests delay process creation, use a few more processors
+else
+    ((maxProcesses = $(getconf _NPROCESSORS_ONLN) * 3 / 2)) # long tests delay process creation, use a few more processors
+fi
```

Comment:
This is minor, but since the logic here is the same and the only difference for NetBSD is the way we get the number of processors, can you just make a variable for number of processors and set that correctly based on OS? That way we will only calculate `maxProcesses` in one place.